sdk/identity/identity/test/node/environmentCredential.spec.ts

```diff
@@ -66,7 +66,7 @@ describe("EnvironmentCredential", function() {
 
     const mockHttpClient = new MockAuthHttpClient();
 
-    const credential = new EnvironmentCredential(mockHttpClient.identityClientOptions);
+    const credential = new EnvironmentCredential(mockHttpClient.tokenCredentialOptions);
```

Comment:
@XiaoningLiu , @jiacfan Please ignore the changes to this file. It was needed to fix the build break in the master branch

Reply:
Thanks for heads up!